java.jdbc-ballerina/Module.md

```diff
@@ -0,0 +1,383 @@
+
+The `dbClient` receives only the database URL.
+
+E.g., The DB client creation for an H2 database will be as follows.
+```ballerina
+jdbc:Client|sql:Error dbClient = new ("jdbc:h2:~/path/to/database");
+```
+
+The `dbClient` receives the username and password in addition to the URL.
+If the properties are passed in the same order as they are defined in the `jdbc:Client`, you can pass them
+without named params.
+
+E.g., The DB client creation for an H2 database will be as follows.
+```ballerina
+jdbc:Client|sql:Error dbClient = new ("jdbc:h2:~/path/to/database", 
+                            "root", "root");
+```
+
+The `dbClient` uses the named params to pass all the attributes and provides the `options` property in the type of
+[jdbc:Options](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/records/Options),
+and also uses the unshared connection pool in the type of
+[sql:ConnectionPool](https://docs.central.ballerina.io/ballerina/sql/latest/records/ConnectionPool).
+For more information about connection pooling, see the [`sql` module](https://docs.central.ballerina.io/ballerina/sql/latest).
+
+E.g., The DB client creation for an H2 database will be as follows.
+```ballerina
+jdbc:Client|sql:Error dbClient = new (url =  "jdbc:h2:~/path/to/database",
+                             user = "root", password = "root",
+                             options = {
+                                 datasourceName: "org.h2.jdbcx.JdbcDataSource"
+                             },
+                             connectionPool = {
+                                 maxOpenConnections: 5
+                             });
+```
+
+The `dbClient` receives some custom properties within the
+[jdbc:Options](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/records/Options),   
+and those properties will be used by the defined `datasourceName`.
+As per the provided example, the `org.h2.jdbcx.JdbcDataSource` datasource  will be configured with a `loginTimeout`
+of `2000` milliseconds.
+
+E.g., The DB client creation for an H2 database will be as follows.
+```ballerina
+jdbc:Client|sql:Error dbClient = new (url =  "jdbc:h2:~/path/to/database", 
+                             user = "root", password = "root",
+                             options = {
+                                datasourceName: "org.h2.jdbcx.JdbcDataSource", 
+                                properties: {"loginTimeout": "2000"}
+                             });                          
+```
+
+You can find more details about each property in the
+[jdbc:Client](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/clients/Client) constructor.
+
+The [jdbc:Client](https://docs.central.ballerina.io/ballerinax/java.jdbc/latest/clients/Client) references
+[sql:Client](https://docs.central.ballerina.io/ballerina/sql/latest/clients/Client) and
+all the operations defined by the `sql:Client` will be supported by the `jdbc:Client` as well.
+
+#### Connection Pool Handling
```

Comment:
This is essentially the same as SQL and leads to duplication of samples, cant we point to SQL database API docs and say all functions supported by SQLClient is supported?

Reply:
Ack